extra/stoatLordCardStuff.js

```diff
@@ -1,7 +1,7 @@
 const fetch = require("node-fetch")
 
 var imfJson = {
-    ruleset: "Stoat Lord Card Thingymagik",
+    ruleset: "Custom TCG Inscryption",
```

Comment:
Name should stay

Reply:
Its not called that tho T_T

Reply:
But it should stay nonetheless. Usually I don't change code name after being releases.